src/pages/quizzes.tsx

```diff
@@ -95,14 +96,14 @@ const QuizzesHubPage = ({ data }: PageProps<Queries.QuizzesHubPageQuery>) => {
       </Box>
 
       <QuizzesHubContext.Provider value={contextState}>
-        <QuizzesModal isOpen={isOpen} onClose={onClose}>
+        <Modal isOpen={isOpen} setIsOpen={onClose}>
```

Comment:
I think we should use directly the Modal component instead of updating the QuizzesModal. Because we have set another prop for the isOpen & setIsOpen prop of the custom Modal & we can add the value where the QuizzesModal component is being rendered. What do you think?

Reply:
@ashiskumar-1999 it's more about the custom styling that is added to this instance of the modal and keeping that styling in isolation. In particular, there is conditional rendering of the background color and the use of the size prop based on screen size.

Also it would be easier to continue handling the background style in `QuizzesModal` than trying to do the logic in a variant for something primitive. That's not how a theme variant should be applied.